README.md

```diff
@@ -0,0 +1,15 @@
+*Build go sdk*
+```
+go get github.com/hyperledger/sawtooth-sdk-go
+cd $GOPATH/src/github.com/hyperledger/sawtooth-sdk-go
+go generate
+```
+Docker instructions
+```
+cd sawtooth-sdk-go/docker
+docker build . -f sawtooth-build-go-protos -t sawtooth-build-go-protos
+cd ..
+docker run -v $(pwd):/project/sawtooth-sdk-go sawtooth-build-go-protos
```

Comment:
Update the above commands when you rename `sawtooth-build-go-protos` to `Dockerfile`.

Also, wrt git hygiene, this commit comes before the "Add Dockerfile" commit, but references content in that commit. When possible, it is easier on reviewers if you order commits so that things you refer to in one commit have already been added in a previous. This enables reviewing commits one at a time.